hs_app_netCDF/models.py

```diff
@@ -185,26 +337,20 @@ class Meta:
 processor_for(NetcdfResource)(resource_processor)
 
 
-# define the netcdf metadata
-class NetcdfMetaData(CoreMetaData):
+class NetCDFMetaDataMixin(models.Model):
+    """This class must be the first class in the multi-inheritance list of classes"""
     variables = GenericRelation(Variable)
     ori_coverage = GenericRelation(OriginalCoverage)
```

Comment:
I wonder about this name. Is the intent to allow people to edit this to create non-original coverages?